external-docs/docs/guides/pivoted-shape.md

```diff
@@ -0,0 +1,105 @@
+            "unit": {
+                "label": "Count"
+            },
+            "measure": {
+                "label": "Number of Stores"
+            }
+        },
+        "Status": {
+            "type": "attribute"
+        }
+    }
+}
+```
+
+## Multiple Measures
+
+Incorporating multiple measures into the pivoted shape can be achieved by defining [unit](../glossary/index.md#unit) and [measure](../glossary/index.md#measure) information for **each** `observation` column. This can be configured in two ways:
+
+1. By specifying a `unit` property within the `observation` column definition (as with the "Number of Arthur's Bakes" [observation](../glossary/index.md#observation-observed-value) column).
+2. By associating a separate `unit` column with the relevant `observation` column using the `describes_observations` property (as with the "Revenue Units" [unit](../glossary/index.md#unit) column). See the [Measure and Unit Columns Configuration](./configuration/qube-config.md/##measure-and-unit-columns-configuration) section for more information on the `describes_observation` property.
+
+Similarly, the `describes_observation` property can be used to associate _attributes_ with the relevant _observation_ values, as with the "Number of Stores Status" and "Revenue Status" [attribute](../glossary/index.md#attribute) columns below. See the [Attributes configuration](./configuration/qube-config.md/#attributes-configuration) section for more information on the `describes_observation` property.
+
+| Year | Location | Number of 'Arthur's Bakes' | Number of Stores Status | Revenue | Revenue Units  | Revenue Status |
+|:-----|:---------|---------------------------:|:------------------------|--------:|:---------------|:---------------|
+| 2022 | London   |                         35 | Provisional             |      25 | GBP (Sterling) | Provisional    |
+| 2021 | Cardiff  |                         26 | Final                   |      18 | GBP (Sterling) | Final          |
+
+```json
+{
+    "$schema": "http://purl.org/csv-cubed/qube-config/v1",
+    "title": "'Arthur's Bakes' stores and revenues in UK cities from 2020 to 2022",
+    "description": "The number of 'Arthur's Bakes' stores and store revenues in cities across the UK between 2020 and 2022.",
+    "creator": "https://www.gov.uk/government/organisations/hm-revenue-customs",
+    "publisher": "https://www.gov.uk/government/organisations/hm-revenue-customs",
+    "columns": {
+        "Year": {
+            "type": "dimension"
+        },
+        "Location": {
+            "type": "dimension"
+        },
+        "Number of 'Arthur's Bakes'": {
+            "type": "observations",
+            "unit": {
+                "label": "Count"
+            },
+            "measure": {
+                "label": "Number of Stores"
+            }
+        },
+        "Number of Stores Status": {
+            "type": "attribute",
+            "describes_observations": "Number of Arthur's Bakes"
```

Comment:
You're missing the quotes from the column's title. This shouldn't work.

Reply:
Done